src/frontend/planner_test/tests/testdata/output/array.yaml

```diff
@@ -33,12 +33,12 @@
 - sql: |
     select ARRAY[]::int[];
   logical_plan: |-
-    LogicalProject { exprs: [Array::List(Int32) as $expr1] }
+    LogicalProject { exprs: [Array as $expr1] }
```

Comment:
An array of the correct type (not displayed) is constructed directly, rather than constructing `varchar[]` first followed by a cast (displayed).